Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -1243,7 +1242,7 @@ private void AddFixToResult(FlexMatch flexMatch, SimpleFix simpleFix, Result res
         {
             int argsCount = namedArgumentToIndexMap.Count;
 
-            var arguments = new List<string>(new string[argsCount]);
```

Comment:
>new string[argsCount] [](http://example.com/codeflow?start=45&length=21)

this was creating a string array with argsCount position, that wasn't being used.
Passing the count prevents resizing.